File: factorial_100.c
#include <stdio.h>

int main()
{
    int T, N, count, k;
    long long sum;
    scanf("%d", &T);
    for (int i = 0; i < T; i++)
    {
        scanf("%d", &N);
        sum = 1;
        for (int j = 1; j <= N; j++)
        {
            sum = sum * j;
        }

        printf("%ld", sum);
        //     count = 0;
        //     k = 10;
        //     while (1)
        //     {
        //         if (sum / k == 0)
        //         {
        //             ++count;
        //             k *= 10;
        //         }else{
        //             break;
        //         }
        //     }

        //     printf("%d",count);
        // }

        return 0;
    }

File: factorial_100.py
T = int(input())

for i in range(T):
    N = int(input())

    sum = 1
    j = 1 
    while j <= N:
        sum = sum * j
        j+=1 
    
    count = 0
    k = 10
    while True:
        if sum % k == 0:
            count += 1
            k = k * 10
        else:
            break
    
    print(count)